modules/govuk_cdnlogs/manifests/init.pp

```diff
@@ -31,7 +31,7 @@
 class govuk_cdnlogs (
   $log_dir,
   $govuk_monitoring_enabled = true,
```

Comment:
Do you want to set this to false too, given we're disabling bouncer monitoring?

Reply:
Done!